### DISCUSSION TITLE
Terms of Use

### DISCUSSION THREAD

**Discussion by vulevukusej** [fief-dev/fief#280] [Q&A]
Is there a way to define additional terms of use on the registration page, such that users need to confirm before signing up?

**Comment by frankie567** [comment]
Not currently, but this is something we could add in the future!

In the meantime, you could handle this in your app using a ["private" user field](https://docs.fief.dev/configure/user-fields/#ask-at-update) (i.e. not settable by the user); a flag like `tos_accepted` which would be `False` by default`. Then, in your app, you can check the value of this flag and prompt the user to accept your terms before they can access the app.

**Comment by vulevukusej** [comment]
Thanks, ill try that out !
